source/sync/data-model/update-schema.txt

```diff
@@ -222,10 +222,11 @@ information, see your preferred SDK's migration page.
 
 .. note:: These changes may trigger a resync
 
-   When you add a new required property, we retreive the documents for 
-   the collection and re-insert them into App Services to get the new values. This
-   is expected behavior, but it does cause a temporary halt to propogating 
-   changes while this process is underway.
+   When you add a new required property, we retreive the documents for the 
```

Comment:
so this is just for adding properties, not object types (i.e. note on line 156 doesn't need to be updated also)

Reply:
Correct - it's about making additive changes to a schema. New object types would have new schemas and this wouldn't apply to them.